scripts/gen_java_client.sh

```diff
@@ -0,0 +1,7 @@
+set -e
+
+docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli generate -i /local/src/main/resources/swagger/api-docs.yaml -g java -o /local/codegen_java --api-package org.broadinstitute.dsde.workbench.client.sam.api --model-package org.broadinstitute.dsde.workbench.client.sam.model
+
+cd codegen_java
+
+sbt test
```

Comment:
I'm confused -- why are you re-testing here?

Reply:
the code generator makes tests too... thought I'd run them

Reply:
the codegen_java dir has its own sbt files so this is running those tests, not the top level tests